script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -83,45 +83,45 @@ a hand of unparalleled heat is unleashed towards Arcueid, melting the rubble in
 <Page24>
 "Arcueid―――!"
 In one instant.
-The tides of battle get turned in one instant.
+The tides of battle are reversed in an instant.
```

Comment:
The tides of battle are turned in an instant.

(Tides turning is the metaphor here)